packages/api-client/demo.ts

```diff
@@ -53,11 +53,11 @@ async function createContext(storeEngine: CRUDEngine, apiClient: APIClient, logi
     const {expiration, zuid} = await storeEngine.read(AUTH_TABLE_NAME, AUTH_COOKIE_KEY);
     const cookie = new Cookie(zuid, expiration);
     logger.log(`Found cookie "${zuid}".`);
-    logger.log('Logging in with EXISTING cookie.');
+    logger.log('Logging in with EXISTING cookie...');
     return apiClient.init(loginData.clientType, cookie);
   } catch (error) {
-    logger.log(`Failed to find existing cookie.`, error);
-    logger.log(`Logging in with NEW cookie: ${error.message}`);
+    logger.log(`Failed to find existing cookie: ${error.message}`, error);
```

Comment:
Why log the error twice? This will generate a log like

`Failed to find existing cookie: <error message> Error: <error message>`.